Cargo.toml

```diff
@@ -36,9 +36,10 @@ path = "src/main.rs"
 name = "prettytable"
 
 [dependencies]
-unicode-width = "0.1"
+unicode-width = "0.2"
 term = "0.7"
 lazy_static = "1.4"
 is-terminal = "0.4"
 encode_unicode = "1.0"
+strip-ansi-escapes = "0.2"

```

Comment:
Do we know the MSRV of this and it's dependencies?

Reply:
No, but it's 1.5 years old, so it's definitely small enough to not bother.